pages/common/makepasswd.md

```diff
@@ -0,0 +1,19 @@
+# makepasswd
+
+> Generate and encrypt passwords.
+
+- Generate a random password:
```

Comment:
What are the defaults? I think it would be nice to state it (if it's not too long)>

Reply:
I'm asking that because I don't think it is truly random, it must follow a pattern.

Reply:
Oh right! I added a note about that.

Reply:
Thank you 😄 looks great